Makefile

```diff
@@ -1779,8 +1859,21 @@ nncp_cleanup: ## unconfigured nncp configuration on worker node and deletes the
 
 .PHONY: netattach
 netattach: export INTERFACE=${NNCP_INTERFACE}
-netattach: export VLAN_START=${NETWORK_VLAN_START}
-netattach: export VLAN_STEP=${NETWORK_VLAN_STEP}
+netattach: export INTERFACE_DATA=${NNCP_INTERFACE_DATA}
+netattach: export INTERFACE_MANAGEMENT=${NNCP_INTERFACE_MANAGEMENT}
+netattach: export INTERFACE_EXTERNAL=${NNCP_INTERFACE_EXTERNAL}
+netattach: export INTERNALAPI_VLAN=${NNCP_INTERNALAPI_VLAN}
+netattach: export STORAGE_VLAN=${NNCP_STORAGE_VLAN}
+netattach: export TENANT_VLAN=${NNCP_TENANT_VLAN}
+netattach: export INTERNALAPI_NET=${NNCP_INTERNALAPI_NET}
+netattach: export STORAGE_NET=${NNCP_STORAGE_NET}
+netattach: export TENANT_NET=${NNCP_TENANT_NET}
+netattach: export INTERNALAPI_IP_START=${NETATTACH_INTERNALIP_IP_START}
+netattach: export INTERNALAPI_IP_END=${NETATTACH_INTERNALIP_IP_END}
+netattach: export STORAGE_IP_START=${NETATTACH_STORAGE_IP_START}
+netattach: export STORAGE_IP_END=${NETATTACH_STORAGE_IP_END}
+netattach: export TENANT_IP_START=${NETATTACH_STORAGE_IP_START}
```

Comment:
I think there's a copy paste from STORAGE lines above here...

Reply:
Ah, you mean STORAGE_IP_[START|END]?
With this variables I mean the storage vlan.
http://dani.foroselectronica.es/openstack-tripleo-networking-layout-696/ (Vlan 30)

Reply:
I mean, why do you use STORAGE_IP_* for TENANT_IP_*? Are these not separate networks? I see in your link:

```
StorageNetCidr: 172.17.3.0/24
StorageNetworkVlanID: 30

TenantNetCidr: 172.17.2.0/24
TenantNetworkVlanID: 50
```

Reply:
Aahh, sorry, I'm blind. Indeed is a copy-paste error, will fix it :)